docs/product/alerts/notifications/index.mdx

```diff
@@ -8,7 +8,7 @@ Sentry sends you notifications regarding workflow activities, [release deploys](
 
 - [Workflow](#workflow-notifications): Activity involving user actions and state changes on issues. This includes activities such as issue resolution, assignment, comments, and regressions.
 - [Deploy](#deploy-notifications): When a release you have commits on is deployed.
-- [Quota](#quota-notifications): Approaching quotas, exceeded quotas, and spike protection.
+- [Spend](#spend-notifications): Reserved volume warning, depleted quotas, and on-demand capacity usage notifications.
```

Comment:
Can you confirm the copy? There seems to be a shift towards "pre-paid" instead of "reserved" and "pay as you go" instead of "on-demamd".
Otherwise, LGTM.

Reply:
"Reserved" is fine, and is a concept we want to keep the same in AM2 and AM3. We're not planning on changing that to "Pre-paid".

"On-Demand" is how AM2 customers see things. However for AM3 orgs, we should change this to "Pay As You Go". Since docs should reflect the latest plan (whereas in-app we show based on their plan), this should be updated from "On-Demand" to "Pay As You Go"

Reply:
Sounds good. Does the copy still need to be changed here? 

This pull request updates the current docs for the Spend Notifications ready to be released for AM1 and AM2 organizations.

Reply:
@dashed @brianthi any updates on this?